files/CooperHewitt-Book.ufo/fontinfo.plist

```diff
@@ -126,7 +126,7 @@
 	<key>openTypeOS2VendorID</key>
 	<string>VLLG</string>
 	<key>openTypeOS2WeightClass</key>
-	<integer>705</integer>
+	<integer>300</integer>
```

Comment:
I’d assign 400 for the Book weight. Book often means Regular or slightly thicker than Regular if such weight is already present.